XamlControlsGallery/ControlPages/ListViewPage.xaml.cs

```diff
@@ -56,6 +85,241 @@ private void SelectionModeComboBox_SelectionChanged(object sender, SelectionChan
                 }
             }
         }
+
+        //===================================================================================================================
+        // Drag/Drop Example
+        //===================================================================================================================
+
+        private void Source_DragItemsStarting(object sender, DragItemsStartingEventArgs e)
+        {
+            // Prepare a string with one dragged item per line
+            StringBuilder items = new StringBuilder();
+            foreach (Contact item in e.Items)
+            {
+                if (items.Length > 0) { items.AppendLine(); }
+                if (item.ToString() != null)
+                {
+                    // Append name from contact object onto data string
+                    items.Append(item.ToString() + " " + item.Company);
+                } 
+            }
+            // Set the content of the DataPackage
+            e.Data.SetText(items.ToString());
+
+            e.Data.RequestedOperation = DataPackageOperation.Move;
+
+        }
+
+        private void Target_DragOver(object sender, DragEventArgs e)
+        {
+            e.AcceptedOperation = DataPackageOperation.Move;
+        }
+
+        private void Source_DragOver(object sender, DragEventArgs e)
+        {
+            e.AcceptedOperation = DataPackageOperation.Move;
+        }
+
+        private async void Target_Drop(object sender, DragEventArgs e)
+        {
+            if (e.DataView.Contains(StandardDataFormats.Text))
+            {
+                DragOperationDeferral def = e.GetDeferral();
+                string s = await e.DataView.GetTextAsync();
+                string[] items = s.Split('\n');
+                foreach (string item in items)
+                {
+
+                    // Create Contact object from string, add to existing target ListView
+                    string[] info = item.Split(" ", 3);
+                    Contact temp = new Contact(info[0], info[1], info[2]);
+                    
+                    // Find the insertion index:
+                    Windows.Foundation.Point pos = e.GetPosition(DragDropListView2.ItemsPanelRoot);
+
+                    // Find height of a random item from the target list view
```

Comment:
This doesn't reflect the code below, which always gets the height of the first item in the list.